gitpod-io/gitpod
# [PAYG - dashboard] Add Gitpod Enterprise callout in New workspace component

## Description


This change adds a Gitpod Enterprise callout under workspace classes option in New workspace section -- only for `gitpod.io` (PAYG) users.

Also, it sends the following event, when someone clicks on `Learn more about Gitpod Enterprise` link & opens it in a browser tab. Links to: https://www.gitpod.io/docs/enterprise

```json
{"anonymousId":"4d4ed229-b209-42aa-b2bd-xxxxx","event":"dashboard_clicked","properties":{"path":"/new","label":"Learn about Gitpod Enterprise","destination":"https://www.gitpod.io/docs/enterprise"}}
```

![image](https://github.com/gitpod-io/gitpod/assets/55068936/23993bf5-1306-466a-bc62-fa695dbe2c28)



## Related Issue(s)

Fixes OPM-602

## How to test


- Open & Login to preview enviroment
- After setup, click on `New workspace`
- Select Workspace classes option and then see under Large workspace class

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - siddhant-of5eaa3d840</li>
	<li><b>🔗 URL</b> - <a href="https://siddhant-of5eaa3d840.preview.gitpod-dev.com/workspaces" target="_blank">siddhant-of5eaa3d840.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - siddhant-opm-602-add-reference-to-enterprise-from-workspace-class-dropdown-gha.25409</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-siddhant-of5eaa3d840%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

      Run the build with werft instead of GHA
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

</details>

<details>
<summary>Installer</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
</details>

/hold
